src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
 // write code here
+
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const horizontalPosition = (wall.clientWidth - spider.clientWidth) / 2;
+
+spider.style.left = horizontalPosition + 'px';
+
+const verticallyPosition = horizontalPosition;
```

Comment:
creating additional variable is redundant)

Reply:
create just one var for example `position` of `shift`  and use it.

Reply:
i did .